rootfs/opt/fluentd/sbin/boot

```diff
@@ -10,6 +10,20 @@ then
   apt-get install -y ruby $BUILD_TOOLS
 fi
 
+# is there a broken symlink in /var/log/containers?
+BROKEN_LINK=$(find /var/log/containers/ -name '*.log' -type l -xtype l -print -quit)
+if [ -n "$BROKEN_LINK" ]; then
+  echo "/var/log/containers contains broken links"
+  # extract the containers/ directory targeted by the broken symlink
+  TARGET_DIR=$(readlink -m "$BROKEN_LINK" | sed 's/\(.*containers\).*/\1/')
```

Comment:
I'd wrap the right side of this statement (the subshell) with double quotes just to be safe with spaces and all...